morl_baselines/multi_policy/envelope/envelope.py

```diff
@@ -365,6 +365,8 @@ def update(self):
                     "global_step": self.global_step,
                 },
             )
+            if self.per:
```

Comment:
wut

Reply:
I though I was committing on the main branch